admin/templates/admin/admin.html

```diff
@@ -4,6 +4,7 @@
 
 {% block scripts %}
 <script src="{{ url_for('admin_bp.static', filename='js/create_preview.js') }}"></script>
+<script src="{{ url_for('static', filename='js/purify.min.js') }}"></script>
```

Comment:
Move to the correct location for third party libraries:  static/lib/purify-3.1.6/